packages/devextreme/testing/tests/DevExpress.ui.widgets/popup.tests.js

```diff
@@ -2877,8 +3011,6 @@ QUnit.module('renderGeometry', {
             title: 'test',
             titleTemplate: () => $('<div>').text('title template'),
             bottomTemplate: () => $('<div>').text('bottom template'),
-            useDefaultToolbarButtons: !options.useDefaultToolbarButtons,
-            useFlatToolbarButtons: !options.useFlatToolbarButtons
```

Comment:
Hm, why should it be removed?

Reply:
Improved here: [f12314916353b33f10d4c7ee4c99a008bcb9d6a4]